src/app/select-session/page.tsx

```diff
@@ -0,0 +1,228 @@
+/**
+ * @author David Zhu <dzhu292@aucklanduni.ac.nz>
+ */
+
+"use client";
+
+import Heading from "@/components/Heading/Heading";
+import Button from "@/components/Button/Button";
+import { useEffect, useState, useRef, ChangeEvent } from "react";
+import { CgProfile } from "react-icons/cg";
+import { AiFillCaretDown } from "react-icons/ai";
+
+import SessionCard from "@/components/SessionCard/SessionCard";
+import { SessionCardStatus } from "@/components/SessionCard/SessionCardStatusEnum";
+import SessionCardProps from "@/components/SessionCard/SessionCardProps";
+import ScrollShadow from "@/components/ScrollShadow";
+import { twJoin } from "tailwind-merge";
+import { useQuery } from "@tanstack/react-query";
+import type { GameSession } from "@prisma/client";
+
+export default function SelectSessionPage() {
+  const remainingSessions = 11;
+  const isMember = true;
+  const firstName = "David";
+
+  const { data } = useQuery({
+    queryKey: ['current-sessions'],
+    queryFn: async () => {
+      const response = await fetch("api/gamesession/current", {
+        cache: "no-store",
+      });
+      return await response.json();
+    }})
+
+  const queriedSessions : SessionCardProps[] | undefined = data?.map((session: GameSession) => {
+    const startDate = session.dateTime
+    const endDate = session.dateTime
```

Comment:
What???? These don't exist in GameSession object.

Reply:
lt was in the pre-existing component someone else made and I wasnt sure if I should change it or not

Reply:
Yea need to change it use `session.startDate` and `session.endDate`. Thats the correct values to use